src/qml/imports/Theme/QfSwitch.qml

```diff
@@ -4,41 +4,5 @@ import QtQuick.Controls
 /**
  * \ingroup qml
  */
-SwitchDelegate {
-  property bool small: false
```

Comment:
Why killing the small property? We still want this flexibility instead of having to manually go for a random width on an individual basis.

Reply:
Just checked now, and yep, we need to use SwitchDelegate, otherwise it's misaligned, see:

<img width="218" height="492" alt="image" src="https://github.com/user-attachments/assets/fb7e9425-4896-4f2c-9b40-e2c73ee3f67a" />